frontend/package.json

```diff
@@ -41,8 +44,28 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "start-electron": "export BROWSER=none && react-scripts start",
+    "build-electron": "npm run build && cp -r electron/. build/electron",
+    "package-electron": "electron-builder build --mac --win --linux -c.extraMetadata.main=build/electron/main.js --publish never",
+    "dev-electron": "export ELECTRON_START_URL=http://localhost:3000 && electron .",
     "lint": "eslint -c .eslintrc.yml --ext .js,.ts,.tsx src/"
   },
+  "build": {
```

Comment:
This should be called `build-electron`

Reply:
I'd still like this to be called `build-electron` or `build-desktop`.

Reply:
This is the build specification for the electron app it has to be called build as it is what the electron packager looks for